### PR TITLE
feat(suite-native): Coin Enabling settings screens polishing

### LINKED ISSUE #13884: Coin Enabling - UI polishing
- [x] add top/bottom gradients to network selection
- [x] update texts to latest version
- [x] Check and update title's text color
- [x] android top/bottom screen offsets are off
- [x] on press coin enabling has odd shadows on android

### PR DESCRIPTION
**Pull request by vytick**
**Fixed following issues:**
- [x] add top/bottom gradients to network selection
- [x] update texts to latest version
- [x] Check and update title's text color
- [x] android top/bottom screen offsets are off
- [x] on press coin enabling has odd shadows on android

## Related Issue

Resolve #13884

### CODE REVIEW

File: suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx [hunk 1]
```diff
@@ -40,6 +36,14 @@ const wrapperStyle = prepareNativeStyle<{ isEnabled: boolean }>((utils, { isEnab
     ],
 }));
 
+const wrapperStyle = prepareNativeStyle(_ => ({
+    paddingVertical: 6,
+    paddingHorizontal: 14,
+    gap: 12,
```
Comment (yanascz) [inline suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx:42]: Would it make sense to use `utils.spacings` for these? At least for the `gap` for which a constant may be used.

Also, is the padding correct according to design? I know if was there before...

File: suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
+    const { applyStyle, utils } = useNativeStyles();
     const { translate } = useTranslate();
+
     const enabledNetworkSymbols = useSelector(selectEnabledDiscoveryNetworkSymbols);
     const availableNetworks = useSelector(selectDiscoverySupportedNetworks);
     const isBitcoinOnlyDevice = useSelector(selectIsBitcoinOnlyDevice);
 
-    const showBtcOnly = availableNetworks.length === 1 && isBitcoinOnlyDevice;
+    //testnets can be enabled and we want to show networks that case
+    const showNetworks = !(availableNetworks.length === 1 && isBitcoinOnlyDevice);
```
Comment (yanascz) [inline suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx]:
```suggestion
    const showNetworks = availableNetworks.length > 1 || !isBitcoinOnlyDevice;
```

File: suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx [hunk 3]
```diff
@@ -34,26 +53,42 @@ export const SettingsCoinEnablingScreen = () => {
...
 
     return (
         <Screen
             screenHeader={
-                <ScreenSubHeader
-                    content={translate('moduleSettings.coinEnabling.settings.title')}
-                />
+                <View style={applyStyle(headerWrapperStyle)}>
+                    <ScreenSubHeader
+                        content={translate('moduleSettings.coinEnabling.settings.title')}
```
Comment (yanascz) [inline suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx:64]: Is it possible to use `Translation` component here as well?

Comment (vytick) [reply suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx:64]: unfortunately no

File: suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx:42]: We have a discussion (or two in slack) for the 12 size and I am in favor of doing this for all occurrences at once.

Comment (yanascz) [reply suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx:42]: Yep, realized later that it's the one that's currently missing.

**Comment by yanascz** [comment]
I'm unable to enable/disable the first/last coin when it's close to the top/bottom edge:
image

Is it possible it's caused by the gradient?

**Comment by vytick** [comment]
Yes, that probably is the reason

File: suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx [hunk 4]
```diff
@@ -1,28 +1,47 @@
...
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { hexToRgba } from '@suite-common/suite-utils';
+
+const GRADIENT_HEIGHT = 40;
+
+const gradientBaseStyleParams = prepareNativeStyle<{ bottom: number }>((_, { bottom }) => ({
+    width: '100%',
+    height: GRADIENT_HEIGHT,
+    position: 'absolute',
+    bottom,
```
Comment (yanascz) [inline suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx:27]: This fixes the issue with toggle buttons below the gradient.
```suggestion
    pointerEvents: 'none',
    bottom,
```

Comment (vytick) [reply suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx:27]: Thanks 🙏 , I am about to look into that later today, thanks for testing and suggestions :)

Comment (vytick) [reply suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx:27]: I did it right away in the end:
b53f863ef7

File: suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx (hunk 2 above)
Comment (vytick) [reply suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx]: b53f863ef7

**Review by yanascz — APPROVED** [review]
Looks good! 🚀

**Pull request merged**